.github/workflows/tkg_integration_tests.yaml

```diff
@@ -15,98 +15,6 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  integ-tests-docker:
```

Comment:
instead of deleting the job, can we use conditions to control job execution? like this https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution. All we need to do is to add this `if: false`

Reply:
It's a good idea. Let me have a try.